module/VuFind/src/VuFind/Role/PermissionProvider/SecureCookie.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+/**
+ * Secure cookie permission provider for VuFind.
```

Comment:
I don't think you need to say "for VuFind" in the comment here. It's implied by the context!

Reply:
thanks @demiankatz that is removed in my follow-up